client/core/core.go

```diff
@@ -7046,3 +7047,140 @@ func (c *Core) DeleteArchivedRecords(olderThan *time.Time, matchesFile, ordersFi
 	}
 	return nil
 }
+
+// AccelerateOrder will use the Child-Pays-For-Parent technique to accelerate
+// the swap transactions in an order.
+func (c *Core) AccelerateOrder(pw []byte, oidB dex.Bytes, newFeeRate uint64) (string, error) {
+	_, err := c.encryptionKey(pw)
+	if err != nil {
+		return "", fmt.Errorf("Accelerate password error: %w", err)
+	}
+
+	tracker, swapCoinIDs, accelerationCoins, changeCoinID, requiredForRemainingSwaps, err := c.orderAccelerationParameters(oidB)
+	if err != nil {
+		return "", err
+	}
+
+	newChangeCoin, txID, err :=
+		tracker.wallets.fromWallet.accelerateOrder(swapCoinIDs, accelerationCoins, changeCoinID, requiredForRemainingSwaps, newFeeRate)
+	if err != nil {
+		return "", err
+	}
+
+	tracker.metaData.ChangeCoin = order.CoinID(newChangeCoin.ID())
+	tracker.metaData.AccelerationCoins = append(tracker.metaData.AccelerationCoins, tracker.metaData.ChangeCoin)
+	tracker.coins[newChangeCoin.ID().String()] = newChangeCoin
```

Comment:
I think the order tracker mutex should be held for the entirety of this, because something may try to touch the change coin in between.